File: hello_flutter/lib/models/UserValidatorModel.dart
class UserValidatorModel {
  bool? usuarioNaoExiste;
  bool? senhaDiferente;

  UserValidatorModel({
    required this.usuarioNaoExiste,
    required this.senhaDiferente,
  });

  factory UserValidatorModel.fromJson(Map<String, dynamic> json) {
    return UserValidatorModel(
      usuarioNaoExiste: json['usuarioNaoExiste'] as bool?,
      senhaDiferente: json['senhaDiferente'] as bool?,
    );
  }

  factory UserValidatorModel.fromJsonUnique(dynamic json) {
    return UserValidatorModel(
      senhaDiferente: json['senhaDiferente'] as bool?,
      usuarioNaoExiste: json['usuarioNaoExiste'] as bool?,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'senhaDiferente': senhaDiferente,
      'usuarioNaoExiste': usuarioNaoExiste,
    };
  }
}

File: hello_flutter_api/hello_flutter_api_dll/Users/UserValidatorModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Google.Cloud.Firestore;
using Newtonsoft.Json;

   public class UserValidatorModel
    {

        public UserValidatorModel(bool usuarioNaoExiste, bool senhaDiferente)
        {
            this.senhaDiferente = senhaDiferente;
            this.usuarioNaoExiste = usuarioNaoExiste;
        }

        [JsonProperty("senhaDiferente")]
        public bool senhaDiferente
        {get;set;}

        [JsonProperty("usuarioNaoExiste")]
        public bool usuarioNaoExiste
        {get;set;}
    }
